pkg/controller/persistentvolume/types.go

```diff
@@ -25,6 +25,13 @@ import (
 	"k8s.io/kubernetes/pkg/client/cache"
 )
 
+const (
+	// A PV created specifically for one claim must contain this annotation in order to bind to the claim.
+	// The value must be the name of the claim being bound to.
+	// This is an experimental feature and likely to change in the future.
+	createdForKey = "provisioning.experimental.kubernetes.io/createdFor"
```

Comment:
good point.  i also saw (after submitting the recent name change) that Tim called for dashes instead of camelCasing in the API doc.   

Key value updated to reflect PV specifically and it uses hot looking dashes instead of the old and busted camelCase.